pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/player/compose/TvPlayerView.kt

```diff
@@ -26,19 +32,32 @@ fun TvPlayerView(
     modifier: Modifier = Modifier
 ) {
     val visibilityState = rememberDelayedVisibilityState(player = player, visible = true)
-    ToggleableBox(
```

Comment:
Should we document that `ToggleableBox` shouldn't be used on TV?

Reply:
Shouldn't we not remove this?